gunicorn/config.py

```diff
@@ -627,11 +627,11 @@ class WorkerClass(Setting):
         A string referring to one of the following bundled classes:
 
         * ``sync``
-        * ``eventlet`` - Requires eventlet >= 0.9.7 (or install it via 
+        * ``eventlet`` - Requires eventlet >= 0.9.7 (or install it via
```

Comment:
I know that your editor did this automatically, but it would be nice not to include cosmetic fixes into this PR. Feel free to submit them as a separate PR.